pkg/libdevfile/libdevfile.go

```diff
@@ -114,6 +114,10 @@ func getDefaultCommand(devfileObj parser.DevfileObj, groupType v1alpha2.CommandG
 		}
 	}
 
+	// if there is only one command of a given group kind, use it as default
+	if len(commands) == 1 {
+		return commands[0], true, nil
+	}
```

Comment:
Can we move this block before the `for` loop (even before L104, where `defaultCmds` is declared and initialized`)? It would avoid iterating for nothing if there is only one command.